README.md

```diff
@@ -8,6 +8,14 @@ An Ansible role that deploys a multi-user [AiiDA lab](aiidalab.materialscloud.or
 
 `ansible-galaxy install marvel-nccr.aiidalab_server`
 
+Note: By default, JupyterHub will use the `aiidalab-docker-stack:latest` image.
+Don't forget to tag the image accordingly after pulling the latest version from Dockerhub:
+
+```
+docker pull aiidalab/aiidalab-docker-stack:latest
+docker tag aiidalab/aiidalab-docker-stack:latest aiidalab-docker-stack:latest
```

Comment:
I suggest specifying the `[aiidalab_server_docker_stack](https://github.com/aiidalab/ansible-role-aiidalab-server/blob/99bdcbd72eb09aceab65c23f30a68d76478a8116/defaults/main.yml#L17)` variable instead.

Reply:
you're right, I should just refer to the variables.

(this text was moved out from the README of the aiidalab-docker-stack repo)